<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IoT_Node_GUIClass</class>
 <widget class="QMainWindow" name="IoT_Node_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IoT_Node_GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="AddItem">
        <property name="text">
         <string>AddItem</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startSim">
        <property name="text">
         <string>StartSim</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="StopSim">
        <property name="text">
         <string>PauseSim</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Clear">
        <property name="text">
         <string>Stop and Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="HideLog">
        <property name="text">
         <string>HideLog</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ShowLog">
        <property name="text">
         <string>ShowLog</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setField">
        <property name="text">
         <string>Set Field</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setRange">
        <property name="text">
         <string>Set Range</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="ConsensusSpeedSlider">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="InteractiveView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>InteractiveView</class>
   <extends>QGraphicsView</extends>
   <header>interactiveview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="IoT_Node_GUI.qrc"/>
 </resources>
 <connections/>
</ui>
